src/apps/loan-list/materials/stackable-material/material-status.tsx

```diff
@@ -59,6 +59,7 @@ const MaterialStatus: FC<MaterialStatusProps> = ({
         <div className="list-reservation__deadline">
           {dueDate && (
             <StatusBadge
+              showBadgeWithoutDueDate={false}
```

Comment:
Normally with boolean props you default to false and then add it if it should be true.
So you could flip the logic around and write:
```typescript
<StatusBadge showBadgeWithDueDate />
```

And then:
```typescript
const StatusBadge: FC<StatusBadgeProps> = ({
  badgeDate,
  warningText,
  showBadgeWithDueDate = false,
  dangerText,
  infoText,
```
and so forth...